### ISSUE TITLE
`Query` not being extended

### ISSUE THREAD

**Issue by MrSaints** [graphql-python/graphene-federation#25]
It seems Graphene Federation has stopped returning an SDL with `extend type Query`, and instead returns `type Query`. I'm not sure if this is intentional, but it is not compatible with some implementations of Federation Gateway. 

See diff: https://github.com/graphql-python/graphene-federation/pull/4/files#diff-3111b5ad3ddd7af276c62d4c15988d6c6f3ee13f1b6eaa87725f4fe94cc65867R42

Other Federation implementation seems to include this, e.g. gqlgen (Go): https://github.com/99designs/gqlgen/blob/2d8673a691deffe6ddd1f4d5f013a52dc91aef91/plugin/federation/federation.go#L223

Apollo [docs](https://www.apollographql.com/docs/federation/subgraph-spec/) seems to suggest it too, but to be honest, it's not too clear. Let me know what y'all think.

**Comment by erikwrede** [comment]
Hey there, which Federation version are you using?

**Comment by MrSaints** [comment]
@erikwrede I'm still on V1.

**Comment by erikwrede** [comment]
AFAIK the `extend` is optional on `type Query` in Federation v2. However, we can surely add it back for both federation versions, as this should not break things.  /cc @patrick91

**Comment by patrick91** [comment]
> AFAIK the `extend` is optional on `type Query` in Federation v2. However, we can surely add it back for both federation versions, as this should not break things. /cc @patrick91

I'd only add it for version 1, as it might be removed in future 😊

**Comment by erikwrede** [comment]
Makes sense. @MrSaints it might be worthwhile to open a bug report at your specific Gateway implementation. Which one are you using? 😊

**Comment by MrSaints** [comment]
We're using Mercurius. I believe they have one opened already tbf, so this isn't totally Graphene Federation's problem. 

https://github.com/mercurius-js/mercurius-gateway/issues/23
